static/app/components/events/interfaces/nativeFrame.tsx

```diff
@@ -151,8 +147,7 @@ function NativeFrame({
 
   const contextLine = (frame?.context || []).find(l => l[0] === frame.lineNo);
   const hasStacktraceLink = frame.inApp && !!frame.filename && (isHovering || expanded);
-  const hasInFrameFeature = hasStacktraceLinkInFrameFeature(organization);
-  const showStacktraceLinkInFrame = hasStacktraceLink && hasInFrameFeature;
+  const showStacktraceLinkInFrame = hasStacktraceLink;
```

Comment:
nit: I this this variable can just be removed since it mimics another value in the same scope